share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
+ |                    |      |                    |                    |
+FreeBSD 15.0-CURRENT  |  NetBSD-current  OpenBSD-current  DragonFly-DEVELOPMENT
  |                    |      |                    |                    |
  v                    v      v                    v                    v
 
 Time
 ----------------
 
-Time tolerance +/- 6 months, depending on which book/article you read; if it
-was the announcement in Usenet or if it was available as tape.
+Time tolerance on 4.3BSD and earlier releases can be as large as six months due
+to the uncertainty of historical records; the early release to special groups;
+and the then slowness of tape duplication. Articles and books from the period
+have different dates, since they relied on different sources: newest file in
+the distribution vs public announcement vs first available tape.
 
-[44B] McKusick, Marshall Kirk, Keith Bostic, Michael J Karels,
-	and John Quarterman. The Design and Implementation of
-	the 4.4BSD Operating System.
-[APL] Apple website [https://www.apple.com/macosx/]
+[44B]  McKusick, Marshall Kirk, Keith Bostic, Michael J Karels, and John
+	Quarterman. The Design and Implementation of the 4.4BSD Operating System.
+[APL]  Apple website [https://www.apple.com/macosx/]
 [BSDI] Berkeley Software Design, Inc.
-[DFB] DragonFlyBSD Project, The.
-[DOC] README, COPYRIGHT on tape.
-[FBD] FreeBSD Project, The.
-[KB]  Keith Bostic. BSD2.10 available from Usenix. comp.unix.sources,
-	Volume 11, Info 4, April, 1987.
-[KKK] Mike Karels, Kirk McKusick, and Keith Bostic. tahoe announcement.
+[DFB]  DragonFlyBSD Project, The.
+[DOC]  README, COPYRIGHT on tape.
+[FBD]  FreeBSD Project, The.
+[KB]   Keith Bostic. BSD2.10 available from Usenix. comp.unix.sources, Volume 11,
+	Info 4, April, 1987.
+[KKK]  Mike Karels, Kirk McKusick, and Keith Bostic. tahoe announcement.
 	comp.bugs.4bsd.ucb-fixes, June 15, 1988.
-[KSJ] Michael J. Karels, Carl F. Smith, and William F. Jolitz.
-	Changes in the Kernel in 2.9BSD. Second Berkeley Software
-	Distribution UNIX Version 2.9, July, 1983.
-[NBD] NetBSD Project, The.
-[OBD] OpenBSD Project, The.
-[QCU] Salus, Peter H. A quarter century of UNIX.
-[SMS] Steven M. Schultz. 2.11BSD, UNIX for the PDP-11.
-[TUHS] The Unix Historical Society.  https://minnie.tuhs.org/Unix_History/.
-[USE] Usenet announcement.
-[WRS] Wind River Systems, Inc.
-[dmr] Dennis Ritchie, via E-Mail
+[KSJ]  Michael J. Karels, Carl F. Smith, and William F. Jolitz. Changes in the
+	Kernel in 2.9BSD. Second Berkeley Software Distribution UNIX Version 2.9,
+	July, 1983.
+[NBD]  NetBSD Project, The.
+[OBD]  OpenBSD Project, The.
+[QCU]  Salus, Peter H. A quarter century of UNIX. ISBN 0201547775, EAN
+	9780201547771
+[SMS]  Steven M. Schultz. 2.11BSD, UNIX for the PDP-11.
+[TUHS] The Unix Historical Society. https://minnie.tuhs.org/Unix_History/.
+[USE]  Usenet announcement.
+[WRS]  Wind River Systems, Inc.
+[dmr]  Dennis Ritchie, via E-Mail
```

Comment:
basically the same comments to all these. I don't think they help the reability and are just churn.